frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -83,35 +85,47 @@ export default class Entity extends React.Component<Props> {
         return 'missing';
     }
 
-    selectEntity = (e: SyntheticMouseEvent<HTMLLIElement>) => {
-        // Flow requires that we use `e.currentTarget` instead of `e.target`.
-        // However in this case, we do want to use that, so I'm ignoring all
-        // errors Flow throws there.
-        // $FLOW_IGNORE
-        if (e.target && e.target.classList.contains('status')) {
-            return null;
-        }
-
+    selectEntity = () => {
         this.props.selectEntity(this.props.entity);
     };
 
+    getSiblingEntities = () => {
+        this.props.getSiblingEntities(this.props.entity.pk);
+    };
+
     toggleForBatchEditing = (e: SyntheticMouseEvent<HTMLSpanElement>) => {
         const { entity, isReadOnlyEditor, isTranslator } = this.props;
 
+        e.stopPropagation();
```

Comment:
Let's move this within the `if` clause: that way we'll select the entity when clicking on the status icon in read-only projects and for non-privileged (incl. non-authenticated) users.